### ISSUE TITLE
The quality of the pseudo-labels in the first few epochs

### ISSUE THREAD

**Issue by xiaona123456** [LiheYoung/UniMatch#65]
When training a new field of the dataset, such as medical, Can you tell me if the pseudo-labels generated in the first few epochs will be of poorer quality, and will they be sufficient for training? I haven't run experiments with my data yet, can you give me some theoretical guidance? Thanks a lot!
```
with torch.no_grad():
        model.eval()

        pred_u_w_mix = model(img_u_w_mix).detach()
        conf_u_w_mix = pred_u_w_mix.softmax(dim=1).max(dim=1)[0]
        mask_u_w_mix = pred_u_w_mix.argmax(dim=1)
```

**Comment by LiheYoung** [comment]
It is normal that pseudo labels in the early stages are poor. They will gradually improve as the training goes. We also use a confidence threshold to discard uncertain pseudo labels, which are mostly poor.

**Comment by LiheYoung** [comment]
Closing due to inactivity.
